pkg/backup/item_backupper_test.go

```diff
@@ -317,11 +329,16 @@ func TestBackupItemNoSkips(t *testing.T) {
 
 			var snapshotService *arktest.FakeSnapshotService
 			if test.snapshottableVolumes != nil {
-				snapshotService = &arktest.FakeSnapshotService{
-					SnapshottableVolumes: test.snapshottableVolumes,
-					VolumeID:             "vol-abc123",
+				if test.snapshotError != nil {
```

Comment:
This is a bit of a mess - would appreciate some ideas on how to make this clearer

Reply:
See my comment below